test/data/README.md

```diff
@@ -1,13 +1,17 @@
 MICROSOFT PROVIDES THE DATASETS ON AN "AS IS" BASIS. MICROSOFT MAKES NO WARRANTIES, EXPRESS OR IMPLIED, GUARANTEES OR CONDITIONS WITH RESPECT TO YOUR USE OF THE DATASETS. TO THE EXTENT PERMITTED UNDER YOUR LOCAL LAW, MICROSOFT DISCLAIMS ALL LIABILITY FOR ANY DAMAGES OR LOSSES, INLCUDING DIRECT, CONSEQUENTIAL, SPECIAL, INDIRECT, INCIDENTAL OR PUNITIVE, RESULTING FROM YOUR USE OF THE DATASETS.
 
+The datasets are provided under the original terms that Microsoft received such datasets. See below for more information about each dataset.
+
 ### Wikipedia Detox
 
-Redistributing the dataset "wikipedia-detox-250-line-data.tsv" with attribution:
+This dataset is provided under [CC0](https://creativecommons.org/share-your-work/public-domain/cc0/). Redistributing the dataset "wikipedia-detox-250-line-data.tsv" with attribution:
 
 > Wulczyn, Ellery; Thain, Nithum; Dixon, Lucas (2016): Wikipedia Detox. figshare. https://doi.org/10.6084/m9.figshare.4054689
 
 With modifications by taking a sample of rows and reducing rough language.
 
+Original source: https://meta.wikimedia.org/wiki/Research:Detox
```

Comment:
I may make sense to add this inside the `>` quoteblock, like we do for the Breast Cancer dataset below.

```
> Wulczyn, Ellery; Thain, Nithum; Dixon, Lucas (2016): Wikipedia Detox. figshare. [https://doi.org/10.6084/m9.figshare.4054689](https://doi.org/10.6084/m9.figshare.4054689)
>
> Original source: [https://meta.wikimedia.org/wiki/Research:Detox](https://meta.wikimedia.org/wiki/Research:Detox)
```